contrib/examples/apiserver/apiserver.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: service-catalog-apiserver
+  labels:
+    app: apiserver
+spec:
+  containers:
+  - name: apiserver
+    image: apiserver
+    imagePullPolicy: IfNotPresent
+    args: 
+    - --etcd-servers
+    - http://localhost:2379
+    - -v
+    - "10"
+    ports:
+    - name: https
+      containerPort: 6443
+  - name: etcd
+    image: gcr.io/google_containers/etcd:3.0.14
+    command: ["etcd"]
```

Comment:
No, the `gcr.io/google_cotnainers/etcd` image simply runs `sh` by default. The `quay.io/coreos/etcd:v3.0.17`, on the other hand, does run `etcd`. Should I just use that one instead? What are the guidelines regarding the use of external images?